query-model/jpql/src/test/kotlin/com/linecorp/kotlinjdsl/querymodel/jpql/path/PathsTest.kt

```diff
@@ -33,6 +33,22 @@ class PathsTest : WithAssertions {
         assertThat(actual).isEqualTo(expected)
     }
 
+    @Test
+    fun `path with a getter`() {
```

Comment:
```suggestion
    fun `path() with a getter`() {
```
How about changing the test names to something like the code above for consistency?